controllers/routes/routes.js

```diff
@@ -71,7 +87,7 @@ module.exports = function(app) {
     });
   });
 
-  app.get('/timeline', isLoggedIn, function(req, res){
+  app.get('/timeline/:username', function(req, res){
```

Comment:
Do we will need another app.get('/timeline/:username')... with middleware(isLoggedIn) function for the user to view/add/edit their page?